liquibase-standard/src/main/java/liquibase/database/AbstractJdbcDatabase.java

```diff
@@ -180,6 +180,7 @@ public void setPreviousAutoCommit(Boolean previousAutoCommit) {
     @Override
     public final void addReservedWords(Collection<String> words) {
         reservedWords.addAll(words);
+        addMySQLReservedWordIfApplicable("MANUAL");
```

Comment:
@MalloD12  Sorry, I just noticed that this was added to AbstractJdbcDatabase.java . Could you override this method in MysqlDatabase class and move this logic there?

Reply:
Do you think it's wrong to have this logic there? Even if it's only applicable for `MySQL`? Do you mean overriding `addReservedWords()` this is a final method, so I wouldn't be able to override it for `MySQL` unless we want to remove the `final` keyword, but I think that would force us to implement this method for other DBs as well.

Reply:
We can remove the final here as it was added only to [make sonar happy ](https://github.com/liquibase/liquibase/blob/3c57492ddc779de35b6e894e2b779fcdfa8b9bbe/liquibase-standard/src/main/java/liquibase/database/core/MariaDBDatabase.java#L21)  (and that's strange) . 
The main reason to move it to `MysqlDatabase` is to have  it in a place that makes more sense to find it later: it is confusing to find a `Mysql` reserved word in the super class if you already have a specialized class for Mysql, right?

Reply:
Hey @filipelautert,

Just so you know- I have updated this PR as suggested. With changes made, there were some test failures because of `PERIOD` reserved word for MariaDB. I was looking for this word, and didn't see it's a `MySQL` reserved  word, I only found it's a reserved word if MariaDB is used in `Oracle` mode but it seems that's only applicable for v10.3.6 (see [here](https://mariadb.com/kb/en/reserved-words/)).

Do you think it's fine if we proceed this way?

Thanks,
Daniel.

Reply:
I found that we missed a call to super in this refactoring - that's why the tests were failing. I added the call and reverted the tests, now it should build fine. Thanks for the refactoring!

Reply:
ohh true, my bad! Thanks for noticing it!